docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2138,3 +2138,50 @@ Result:
 
 -   [tcp_port](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-tcp_port)
 
+## currentRoles {#current-roles}
+
+Returns the names of the roles which are current for the current user. The command [SET ROLE](../../sql-reference/statements/set-role.md#set-role-statement) could be used to change the current roles, by default the current roles are returned by the function [defaultRoles](#default-roles) called when the current user logins.
```

Comment:
Instead of `by default the current roles are returned by the function [defaultRoles](#default-roles) called when the current user logins` please write `if the command `SET ROLE` hasn't been used the function currentRoles() returns the same as `defaultRoles()`.